tests/extra_lines.crt

```diff
@@ -12,8 +12,3 @@ KQjjlfTQ5A052QeXfgTyO9vv1S6MRIi7qgiaEv49cGXnJv/TWbySkMKObPMUApjg
 6z8PqcxuShew5FCTkNvwhABFPiyu0fUj3e2FEPHfsBu76jz4ugtmhUqjqhzwFY9c
 tnWRkkl6J0AjM3LnHOSgjNIclDZG
 -----END CERTIFICATE-----
-
-
-
-
-
```

Comment:
this file is formatted like this on purpose

Reply:
is this just the one, or are there any others to watch out for?